.devcontainer/generate-dev-containers.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# For each of our platforms, generate a devcontainer.json file
+
+PLATFORMS=( "Jammy-arm64" "Jammy-x86_64" "Bionic-x86_64" "Bionic-arm64" "Centos7-x86_64" "openSUSE15-x86_64" "RHEL8-x86_64" "RHEL9-arm64" "RHEL9-x86_64")
+
+for PLATFORM in "${PLATFORMS[@]}"
+do
+    echo "Generating devcontainer.json for $PLATFORM"
+
+    PLATFORM_LOWERCASE=$(echo "$PLATFORM" | tr '[:upper:]' '[:lower:]')
+    PLATFORM_DIR="dev-$PLATFORM_LOWERCASE"
+
+    # Create directory for platform if it doesn't exist
+    if [ ! -d $PLATFORM_DIR ]; then
+        mkdir -p $PLATFORM_DIR
+    fi
```

Comment:
```suggestion
    mkdir -p $PLATFORM_DIR
```
you shouldn't need the `if` clause now that you're using `-p` (assuming that's just there to prevent errors when the directory already exists).

Reply:
Duh! Thanks for catching that.